tgui/packages/tgui/interfaces/CargoConsole.js

```diff
@@ -223,7 +223,7 @@ const CataloguePane = (_properties, context) => {
           {cratesToShow.map((c) => (
             <Table.Row key={c.name}>
               <Table.Cell bold>
-                {c.name} ({c.cost} Credits)
+                {c.name} ({c.get_cost()} Credits)
```

Comment:
Ya cant access .dm functions from inside tgui

Reply:
Any idea how to do this better?

Reply:
It appears you've already done it on the .dm side, all you'll need to do is return this .js file to its original form and it should work fine